Makefile

```diff
@@ -4,13 +4,23 @@ GENERATE_CONFIG_PATH        ?= generate/config.yaml
 GENERATE_TEMPLATES_PATH     ?= generate/templates
 OUTPUT_PATH                 ?= .
 
-.PHONY: regen bootstrap generate format tidy clean format-readme
+.PHONY: regen bootstrap delete-generated generate format tidy clean format-readme
 
-regen: bootstrap generate format tidy clean
+regen: bootstrap delete-generated generate format tidy clean
 
 bootstrap:
 	go install mvdan.cc/gofumpt@latest
 
+delete-generated:
```

Comment:
Not blocking 

Does this overlap with the clean cmd? (Should it?)

Reply:
Good question, it doesn't. The `clean` target is more of a post-generation cleanup. I was trying to remove it with an entry to `.openapi-generator-ignore` but it seems like the ignore file does not respect this.